test.py

```diff
@@ -84,9 +86,9 @@ def test_basic_app(self):
             ):
                 print(line)
 
-        assert len(diff_files) == 0
-        assert len(left_only_files) == 0
-        assert len(right_only_files) == 0
+        assert len(diff_files) == 0, diff_files
```

Comment:
is there a reason you got rid of these?

Reply:
I didn't got rid of those, I added them as the second parameter of the assert. When the assert fails it prints the second parameter, giving a bit extra info of what failed

Reply:
oopth, read the diff backwards